ush/check_yaml_keys.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python3
```

Comment:
we shouldn't use this anymore, global is migrating to `wxflow` YAML tools. Ask @aerorahul for details

Reply:
@CoryMartin-NOAA  Thanks a lot for your comments/suggestions in such a prompt way.  
I would dig more on this and talk to @aerorahul on this .